sde_collections/admin.py

```diff
@@ -6,7 +6,21 @@
 from .models.candidate_url import CandidateURL, ResolvedTitle
 from .models.collection import Collection, WorkflowHistory
 from .models.pattern import DivisionPattern, IncludePattern, TitlePattern
-from .tasks import import_candidate_urls_from_api
+from .tasks import fetch_and_update_full_text, import_candidate_urls_from_api
+
+
+@admin.action(description="Import candidate URLs from LRM Dev Server with Full Text")
+def fetch_full_text_lrm_dev_action(modeladmin, request, queryset):
+    for collection in queryset:
+        fetch_and_update_full_text.delay(collection.id, "LRM_DEV")
+    modeladmin.message_user(request, "Full text fetched and updated from LRM_DEV successfully.")
```

Comment:
Please update this message to be more accurate. Something like `f"Started importing URLs from the API for: {collection_names} from {server_name.title()}"`

Reply:
The suggested change has been incorporated